mp_ephem/ephem.py

```diff
@@ -979,9 +980,9 @@ def from_string(cls, comment):
                          y=values[5].strip(),
                          comment=comment_string.strip())
         except Exception as e:
-            logging.error(values)
-            logging.error(comment)
-            logging.error(str(e))
+            logging.debug(values)
```

Comment:
could these be warning messages?  Then the user can turn off.. making debug is maybe not what you want.